eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -40,7 +47,7 @@ the updated actions to
 When set to \fBcheck\fP, the tool will check to see if all actions are applied (e.g., that
 applications that should have been installed are installed and every app that
 should have been uninstalled is not installed).
-.\"
+\".
```

Comment:
Nitpick: This should be in a separate commit.

Reply:
I don't understand this change or know why it's in there, so I guess I'll take it out.